examples/train_model.py

```diff
@@ -33,14 +33,17 @@
 import math
 import os
 
-def run_eval(agent, opt, datatype, still_training=False):
+def run_eval(agent, opt, datatype, still_training=False, max_exs=-1):
```

Comment:
oh I put it there to make it explicit, since even if you set max-valid on the command line you probably don't want to use it at the very end when you're doing the final validation/testing (you can see below it's only used for one `run_eval` but not the other two).

I guess that coincides with the use of `still_training`, so I could interpret from that instead, this is just more explicit